apps/prairielearn/src/lib/externalGrader.ts

```diff
@@ -61,10 +60,6 @@ export function init(): void {
   }
 }
 
-export async function beginGradingJobs(grading_job_ids: string[]): Promise<void> {
-  await async.each(grading_job_ids, beginGradingJob);
-}
-
```

Comment:
This function was only used in assessment.ts, but the call was removed there since it was unnecessary. As such, this function becomes unnecessary as well.